storage-operator/README.md

```diff
@@ -0,0 +1,19 @@
+# Storage Operator
+
+## How to run the operator locally
+
+1. Connect to the bootstrap node
+2. Copy the /etc/kubernetes/admin.conf from the bootstrap node to your local
+   machine.
+3. Get the "public" IP of the Salt API server (the one that use the port 4507):
+
+    `kubectl --kubeconfig /etc/kubernetes/admin.conf -n kube-system describe svc salt-master | grep 'Endpoints'`
+
+4. Start the operator on your machine
+
+```shell
+export KUBECONFIG=<path-to-the-admin.cong-you-copied-locally>
+export METALK8S_SALT_MASTER_ADDRESS=http://<IP-OF-SALT-API-WITH-PORT>
+cd storage-operator
```

Comment:
Shouldn't we register the CRD with the API server first?

Reply:
Yes indeed.

I didn't write it because soon™ the CRD will be registered when installing MetalK8s so it won't be necessary.

Meanwhile, I can add it (and remove it when it's actually implemented)